### PR TITLE
MODSOURCE-76 - Added 'externalEntityId' field to recordDto and recordModel schemas

### PR DESCRIPTION
**Pull request by RuslanLavrov**

### CODE REVIEW

File: examples/mod-source-record-storage/recordCollection.sample [hunk 1]
```diff
@@ -447,6 +447,9 @@
           ]
         }
       },
+      "externalIdsHolder": {
+        "instanceId": "54cc0262-76df-4cac-acca-b10e9bc5c79a"
```
Comment (Igor-Gorchakov) [inline examples/mod-source-record-storage/recordCollection.sample]: Please rename it from "instanceId" into "inventoryInstanceId" to be able to differ instance from mod-inventory/mod-marccat

File: schemas/common/externalIdsHolder.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Contains identifiers of external entities (instance, holding)",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "instanceId": {
```
Comment (Igor-Gorchakov) [inline schemas/common/externalIdsHolder.json]: Please rename it from "instanceId" to "inventoryInstanceId" to be able to differ instance from mod-inventory/mod-marccat

File: schemas/mod-source-record-storage/recordModel.json [hunk 3]
```diff
@@ -63,6 +68,7 @@
     "snapshotId",
     "matchedId",
     "recordType",
+    "externalIdsHolder",
```
Comment (KaterynaSenchenko) [inline schemas/mod-source-record-storage/recordModel.json]: make it not required pls

File: schemas/common/externalIdsHolder.json (hunk 2 above)
Comment (KaterynaSenchenko) [reply schemas/common/externalIdsHolder.json]: let's leave it as instanceId for now, it reflects the 'i' subfiled of '999' field.

**Pull request merged**